tests/kernel/mem_protect/userspace/prj.conf

```diff
@@ -4,3 +4,5 @@ CONFIG_INIT_STACKS=y
 CONFIG_APPLICATION_DEFINED_SYSCALL=y
 CONFIG_TEST_USERSPACE=y
 CONFIG_PIPES=y
+CONFIG_TEST_HW_STACK_PROTECTION=n
+CONFIG_COMMON_LIBC_MALLOC_ARENA_SIZE=0
```

Comment:
I don't think this should be in `prj.conf`. Besides, the `kernel.memory_protection.userspace` test case already has this config.